src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/BookBinder.java

```diff
@@ -110,13 +114,23 @@ public String getMachineIdentifier() {
         for (Map.Entry<Enchantment, Integer> entry : ech2.entrySet()) {
             for (Map.Entry<Enchantment, Integer> conflictsWith : enchantments.entrySet()) {
                 if (entry.getKey().conflictsWith(conflictsWith.getKey())) {
-                    conflicts = true;
+                    if (entry.getKey() == conflictsWith.getKey()) { 
+
+                    } else {
```

Comment:
Why is there an empty if-block here?

Reply:
having the same key as between entry and conflictsWith that is overleveled (ex unbreaking 6) still makes conflictsWith = true

Reply:
so that is set their just in case, it can probably be moved into the higher if block

Reply:
that doesn't answer why there's an empty if. Just negate (but also don't create an inner-if for no reason)